### PR TITLE
Add dot converter

### LINKED ISSUE #11: Make a converter for DOT format
Convert SGraph to DOT format https://graphviz.org/doc/info/lang.html

See this as a reference: https://github.com/softagram/sgraph/blob/main/src/sgraph/converters/xml_to_dot.py

Remember to add unit tests.

### PR DESCRIPTION
**Pull request by NotMixu**
Closes #11

### CODE REVIEW

File: src/converters/sgraphToDot.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { SElement } from '../selement';
+import { SGraph } from '../sgraph';
+
+const tab = '\t';
+
+const sgraphToDot = (graph: SGraph): string => {
```
Comment (skis84) [inline src/converters/sgraphToDot.ts:6]: Please add documentation for this function

File: test/sgraphToDot.test.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      }
+      label = "cyclic";
+    }
+  }
+
+  "c1ffc1e928008e0d84a6640e8e16b4359387810f" -> "5041c6f9a5895b3e49d0232bb4d25b258a763eaf" [label="import"];
+  "c1ffc1e928008e0d84a6640e8e16b4359387810f" -> "1c805310968af27bc31604e3e7a1a790dee27595" [label="import"];
+}`;
+
+describe('sgraphToDot', () => {
```
Comment (skis84) [inline test/sgraphToDot.test.ts:33]: There could be a test case with some other model than cyclic_model.xml. Are there any error cases that could occur and should be tested?

**Review by skis84 — APPROVED** [review]
Looks good overall, add some documentation and think if you can come up with more test cases.

**Review by villelaitila — COMMENTED** [review]
Looks good!

**Pull request merged**